core/rawdb/schema.go

```diff
@@ -203,6 +205,11 @@ func blockReceiptsKey(number uint64, hash common.Hash) []byte {
 	return append(append(blockReceiptsPrefix, encodeBlockNumber(number)...), hash.Bytes()...)
 }
 
+// blockBlobsKey = BlockBlobsPrefix + num (uint64 big endian) + hash
+func blockBlobsKey(number uint64, hash common.Hash) []byte {
```

Comment:
Can we write a description here mentioning what exactly are `number` and `hash`?